vividus/src/test/java/org/vividus/runner/VividusInitializationCheckerTests.java

```diff
@@ -45,8 +60,14 @@ public void before()
         when(BeanFactory.getBeanDefinitionNames()).thenReturn(new String[] { BEAN_1, BEAN_2 });
     }
 
+    @After
+    public void after()
+    {
+        TestLoggerFactory.clear();
```

Comment:
Is there a rule for clean-up? Unfortunately lidalia's site not opening

Reply:
JUnit rule is available, but its usage triggers spotbugs violation: URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD